content/v2.0/query-data/guides/scalar-values.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Extract scalar values in Flux
+description: >
+  placeholder
+menu:
+  v2_0:
+    name: Extract scalar values
+    parent:  How-to guides
+weight: 210
+v2.0/tags: [scalar]
+---
+
+Use Flux [stream and table functions](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/)
+to extract scalar values from Flux query output for things like dynamically setting
+variables using query results.
+
+**To extract scalar values from output:**
+
+1. [Extract a table](#extract-a-table)
+2. Use the extracted table and [extract an array of column values](#extract-a-column-from-the-table)
+   or [extract a row as an object](#extract-a-row-from-the-table).
+
+_The samples on this page use the [sample data provided below](#sample-data)._
+
+{{% warn %}}
+#### Current limitations
+- The InfluxDB user interface (UI) does not support raw scalar output, but you can
+  use [`map()`](/v2.0/reference/flux/stdlib/built-in/transformations/map/) to add
+  scalar values to data.
+- The [Flux REPL ](/v2.0/reference/cli/influx/repl) does not currently support
+  Flux stream and table functions (also known as "dynamic queries").
+  See [#15321](https://github.com/influxdata/influxdb/issues/15231).
+
+{{% /warn %}}
+
+## Extract a table
+Flux formats query results as a stream of tables.
+To extract a scalar value, first first use
```

Comment:
Flux formats...as a stream of tables.
1. Use tableFind() to extract the first table in a stream where the group key matches the predicate function.